controllers/apply/expiries/send-expiry-email.js

```diff
@@ -14,18 +14,21 @@ function getSubjectLine(emailType, isBilingual = false) {
     let subjectLine;
     switch (emailType) {
         case 'AFA_ONE_MONTH':
+        case 'UNDER10K_ONE_MONTH':
```

Comment:
Had to add these as additional cases to allow for the three month backlog of existing ones in the database.